migrations/2022-08-09-142755_create_tables_weather_bot/up.sql

```diff
@@ -26,9 +25,22 @@ CREATE TABLE chats (
   id BIGINT,
   user_id BYTEA,
   state client_state DEFAULT 'initial' NOT NULL, -- Initial
-  before_state client_state DEFAULT 'initial' NOT NULL, -- Initial
   selected VARCHAR(80),
   default_city_id INT,
   PRIMARY KEY (id, user_id),
   CONSTRAINT fk_cities FOREIGN KEY(default_city_id) REFERENCES cities(id)
 );
+
+CREATE TABLE forecasts (
+  chat_id BIGINT,
+  user_id BYTEA,
+  city_id INT,
+  cron_expression VARCHAR(80),
+  last_delivered_at TIMESTAMP WITH TIME ZONE,
+  next_delivery_at TIMESTAMP WITH TIME ZONE NOT NULL,
+  update_at TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
what do you think about renaming:

update_at => updated_at
create_at => created_at

also I think updated_at can also have a default value (NOW())

Reply:
Done.